### PR TITLE
add github workflows

### PR DESCRIPTION
**Pull request by CarloLucibello**

### CODE REVIEW

**Comment by CarloLucibello** [comment]
shouldn't github ci be triggered already?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (simeonschaub) [inline .github/workflows/ci.yml:5]: Not sure that's really the problem,  but you could try removing this line.

Comment (CarloLucibello) [reply .github/workflows/ci.yml:5]: looks like there is a widespread problem, It stopped working also on zygote https://github.com/FluxML/Zygote.jl/pull/973

Comment (CarloLucibello) [inline .github/workflows/ci.yml:5]:
```suggestion

```

Comment (CarloLucibello) [reply .github/workflows/ci.yml]: removing this line didn't help

**Comment by DhairyaLGandhi** [comment]
Oops, didn't see this earlier, my apologies. It does trigger now though

**Comment by CarloLucibello** [comment]
good

**Pull request closed without merging**